climada/hazard/test/test_trop_cyclone.py

```diff
@@ -413,29 +414,33 @@ def test_apply_criterion_track(self):
         intensity[3, 3] = 3
         tc = TropCyclone(
             intensity=sparse.csr_matrix(intensity),
-            basin=['NA', 'NA', 'NA', 'NO'],
+            basin=['NA', 'NA', 'NA', 'WP'],
             category=np.array([2, 0, 4, 1]),
             event_id=np.arange(4),
             frequency=np.ones(4) * 0.5,
         )
 
-        tc_cc = tc.apply_climate_scenario_knu(ref_year=2050, rcp_scenario=45)
-        self.assertTrue(np.allclose(tc.intensity[1, :].toarray(), tc_cc.intensity[1, :].toarray()))
-        self.assertTrue(np.allclose(tc.intensity[3, :].toarray(), tc_cc.intensity[3, :].toarray()))
+        tc_cc = tc.apply_climate_scenario_knu(percentile='50',
+                                              rcp_scenario=45,
+                                              target_year=2050)
+        self.assertFalse(
+            np.allclose(tc.frequency[1], tc_cc.frequency[1])
+            )
+        self.assertFalse(
+            np.allclose(tc.frequency[0], tc_cc.frequency[0])
+        )
         self.assertFalse(
-            np.allclose(tc.intensity[0, :].toarray(), tc_cc.intensity[0, :].toarray()))
+            np.allclose(tc.frequency[2], tc_cc.frequency[2])
+            )
```

Comment:
This appears to be a very weak test where one only checks that the values changed, but not that the values are correct. Is this intended?